src/domain/auth/state/AuthenticationReducers.ts

```diff
@@ -1,9 +1,17 @@
 import { reducer as oidcReducer } from 'redux-oidc';
 import { combineReducers } from 'redux';
+import storage from 'redux-persist/lib/storage';
+import { persistReducer } from 'redux-persist';
 
 import backendAuthenticationReducer from './BackendAuthenticationReducer';
 
+const backendPersistConfig = {
+  key: 'backendPersistor',
+  storage,
+  whitelist: ['apiToken'],
```

Comment:
persist the apiToken, prevent overfetching when navigating around app.
When token expired -> apollo error gonna catch the error -> display prompt -> log user out.